documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Improved migration plan in the console
+
+The user interface console has improved the process of creating a migration plan.
+
+.`instance type` and `instance type preferences` have replaced {ocp-name} templates
+
+The `instance type preferences` have replaced {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+Customer Virtual Machine Mapping are custom templates that you can configure for specific migrations, VMs, and users.
+
+.Full support for migration from OVA
+
+{project-short} supports migration from OVA.
+
+.Integration of the `Create` and `Edit` migration plans
+
+The user interface `Create` and `Edit` elements for migration plans now align with the look and feel of the {ocp} web console and displays up-to-date data. These elements now replace Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
```

Comment:
```suggestion
The user interface `Create` and `Edit` elements for migration plans now align with the design of the {ocp} web console and display up-to-date data. These elements replace Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
```

Is it ok to replace "look and feel" with "design"?

Reply:
Defo